Charades_Game_In_JavaScript/Charades Game In JavaScript final blog.md

```diff
@@ -0,0 +1,348 @@
+
+Here we take `time()` for example, and the `word()` is implemented in the exact same way.
+
+1. Hide Texts:
+To select the text boxes, we set it to a variable `var element = document.getElementById("intro-words");`. Then, we would modify the style of this element into `"hide"`. Same for the `"intro-time"` text box.
+
+![](https://i.imgur.com/0NXOPNd.png)
+
+2. Hide Buttons:
+Since there are two buttons under the `but` class, we need to select them with `const button = document.querySelectorAll('.but');`, then access each of them by using `forEach(but => but...);` The button have a style property called visibility, so we could simple make it to "hidden".
+
+![](https://i.imgur.com/VNnoB6d.png)
+
+3. Show the Counter:
+Same as hiding the text boxes, we would select the timer element by its id `"go"`, and use `.classList.toggle` to set it to `"show"`.
+
+
+### **Phase 2: Timer**
+Now we could implement the timer.
+The idea is that the timer will take in the `textContent` of our `"go"` element, and show the modified `textContent` every one second. And we update the `textContent` by keeping a counter and decrement it every second.
+
+##### **in JavaScript:**
+We first create a variable `seconds` to hold the value in the `textContent` of our `"go"` element, which is 5.
+
+![](https://i.imgur.com/itWasch.png)
+
+Then, we would use `setInterval()` function to let the timer count down at each second (=1000 milliseconds), and update the variable `seconds` by using the decrementing operator `seconds--;`, and also, update the `textContent` with the decremented seconds or whatever words you want (like "Ready??" "Go!!").
+
+We use an `if-else if-else` loop to separate conditions on the seconds. The seconds would go into the `else` statement after 5 iterations, and we would `clearInterval(count);` and jump to the corresponding page.
+
+![](https://i.imgur.com/k6gd2dv.png)
+
+Now try out the new feature you got for your limiting time version! 
+
+### **Phase 3: Copy to Both Functions**
+
+##### **in JavaScript:**
+Simply copy and paste all the codes we added in the function `time()` in phase 1 and 2 into the function `word()`. Remember to change the redirecting page into `"card_word.html"`!
+
+Now, try clicking both of the buttons!
+
+
+
+
+## Creating Animated Word Cards
+![](https://i.imgur.com/9XlNjfr.png)
+
+Now it is the exciting part to create flipping cards with words on them!
+Let's start with creating the elements in HTML.
+
+##### **in HTML:**
+The HTML of both versions are exactly the same, besides the different JavaScript they included.
+
+There will be a simple `<div>` holding the card which has a `front-face`, and a `back-face`. As learned from Marina Ferreira's blog, I adapted her procedure to create the flipping card. And in the end, I add a timer to give a reference to the players.
```

Comment:
is there a better way to quote the blog, maybe saying "I adapted a procedure from the original blog to create the flipping card" This way, you aren't using a name. In other places where you use her name, try to introduce the blog in a different way, like I did here.